tests/test-map/package.json

```diff
@@ -2,10 +2,9 @@
   "private": true,
   "name": "@hpcc-js/test-map",
   "dependencies": {
-    "@hpcc-js/common": "^2.17.0",
-    "@hpcc-js/dgrid-shim": "^2.10.2",
+    "@hpcc-js/common": "^2.17.2",
     "@hpcc-js/map": "^2.7.11",
-    "@hpcc-js/util": "^2.7.2",
```

Comment:
removed util but it's used in `map-deck.spec.ts`? Does the inclusion of common allow for this?